Изменила ДЗ№7

NKVol opened the pull request:
Изменила проверку наличия элемента

NKVol committed:
Добавила ДЗ7 и файл описания

NKVol committed:
Подкорректировала файлы

NKVol committed:
Update Selenium2gisTest.py

убрала лишний wait

NKVol committed:
Update Selenium2gisTest.py

Изменила проверку наличия элемента

NKVol committed:
Create requirements.txt

NKVol committed:
Delete TestCase.md

skyline-gleb commented on Selenium2gisTest.py:
```diff
@@ -0,0 +1,33 @@
+        search_bar.send_keys("версаль, торгово-офисный центр")
+        search_bar.submit()
+        search_list = self.driver.find_element_by_class_name("searchResults__list")
+        self.assertTrue("Версаль, торгово-офисный центр" in search_list.text)
```
есть более подходящий assert, поищи

skyline-gleb commented on Selenium2gisTest.py:
```diff
@@ -0,0 +1,33 @@
+        self.driver.get("http://2gis.ru/novosibirsk")
+
+    def test_obj_designation(self):
+
```
пустая строка плохо, антикодстайл

skyline-gleb commented on Selenium2gisTest.py:
```diff
@@ -0,0 +1,33 @@
+        2. Нажать enter
+    Ожидаемый результат:
+        Ссылка на искомый объект появилась в списке результатов поиска(при условии наличия объекта в базе)
+    """
```
прилодила то не к тесты. класс != тест

skyline-gleb commented:
следующие ПР не приму пока они новые, а не переоткрытые. тебе инвайт выслан, прими его, чтоб появились права переоткрывать ПР

NKVol committed:
Update Selenium2gisTest.py

Поменяла assert
Добавила комментарии в setUp и tearDown
Перенесла описание тест кейса в тест
Убрала пробелы

NKVol commented:
Поменяла assert
Добавила комментарии в setUp и tearDown
Перенесла описание тест кейса в тест
Убрала пробелы

skyline-gleb commented on Selenium2gisTest.py:
```diff
@@ -0,0 +1,32 @@
+        self.assertIn("Версаль, торгово-офисный центр", search_list.text)
+
+    def tearDown(self):
+        """ Завершение сессии """
```
не надо таких комментариев, лишняя трата времени

skyline-gleb commented:
:+1:
